### PR TITLE
Upgrade ember-cli and improve tree-shaking

### PR DESCRIPTION
**Pull request by villander**

### CODE REVIEW

File: app/transitions.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export default function(){
+  // Add your transitions here, like:
+  //   this.transition(
+  //     this.fromRoute('people.index'),
+  //     this.toRoute('people.detail'),
+  //     this.use('toLeft'),
+  //     this.reverse('toRight')
+  //   );
+}
```
Comment (ef4) [inline app/transitions.js]: This file doesn't belong in an addon (if it exists at all it should be tests/dummy/app/transitions.js). That may be a bug in the liquid-fire generator.

File: .travis.yml [hunk 2]
```diff
@@ -21,8 +21,8 @@ env:
   matrix:
     # we recommend new addons test the current and previous LTS
     # as well as latest stable release (bonus points to beta/canary)
-    - EMBER_TRY_SCENARIO=ember-lts-2.8
```
Comment (ef4) [inline .travis.yml:24]: Let's keep the lts-2.8 test, there's no reason not to keep supporting it.

File: package.json [hunk 3]
```diff
@@ -1,52 +1,48 @@
 {
   "name": "memory-scroll",
-  "version": "0.9.1",
+  "version": "0.0.0",
```
Comment (ef4) [inline package.json]: we lost our version number here

**Review by ef4 — CHANGES_REQUESTED** [review]
Thanks, I left a couple small bits of feedback.

Also, as an aesthetic comment, I don't personally like `const` for local variables that only happen to not be mutated at the moment. I realize this is an endless bikeshed and I wouldn't block the PR for just that. 

But my thought process is: would it *necessarily* be wrong for some future editor of the code to mutate the variable? If not, then it should really be `let`. `const` is communicating something too strong to future readers.

File: .travis.yml (hunk 2 above)
Comment (villander) [reply .travis.yml:24]: Ok

File: app/transitions.js (hunk 1 above)
Comment (villander) [reply app/transitions.js]: Ok

File: package.json (hunk 3 above)
Comment (villander) [reply package.json]: Ok

**Comment by villander** [comment]
Done @ef4

**Comment by villander** [comment]
Done @ef4

**Comment by villander** [comment]
@ef4 we can do the merge this?

**Pull request merged**